### PR TITLE
Component cmz styling

### LINKED ISSUE #6: Style SolutionForm component
The task here is to style the ``SolutionForm`` component as described by the mockups on #1. Implement CSS using [cmz](https://github.com/joshwnj/cmz). Also, here's an example on using ``cmz``: https://www.youtube.com/watch?v=5GZ2Moor4V4.

### PR DESCRIPTION
**Pull request by diegocasmo**
Closes #6

### CODE REVIEW

File: src/SolutionForm/index.js [hunk 1]
```diff
@@ -1,21 +1,96 @@
 import React from 'react'
+import cmz from 'cmz';
 
-module.exports = props => <div className="component-SolutionForm">
-  <div className='title'>
-    <h1 className='heading'>{props.heading}</h1>
-    <h2 className='subHeading'>{props.subHeading}</h2>
-    <textarea
+const styles = cmz({
```
Comment (diegocasmo) [inline src/SolutionForm/index.js]: I'm not sure if you would rather have a ``styles.js`` file for this CSS. I initially thought about doing so, but it kind of makes sense to have it all in the same place (it's just one component, all CSS belongs to this component, etc).

Let me know what are your thoughts on this.

File: src/SolutionForm/index.js [hunk 2]
```diff
@@ -1,21 +1,103 @@
...
-module.exports = props => <div className="component-SolutionForm">
-  <div className='title'>
-    <h1 className='heading'>{props.heading}</h1>
-    <h2 className='subHeading'>{props.subHeading}</h2>
-    <textarea
+const Util = cmz('SolutionFormUtil', {
+  maxWidth: 'max-width: 740px'
+})
+
+const styles = cmz('SolutionForm', {
```
Comment (joshwnj) [inline src/SolutionForm/index.js]: I've given the module a name, as that will result in more readable classes.

File: src/SolutionForm/index.js (hunk 1 above)
Comment (joshwnj) [inline src/SolutionForm/index.js]: Yep I think it's fine in this case.

**Comment by joshwnj** [comment]
@diegocasmo looking good!

 I made a couple of small adjustments. It'd be good to extract colours out into an object, and also think about whether there are any other common styles that should be refactored out into composition classes (eg. in the "Util" module)

File: src/SolutionForm/index.js (hunk 2 above)
Comment (diegocasmo) [inline src/SolutionForm/index.js]: Took me forever to realize this is only possible with the newer version (I had ``2.0.1`` and it didn't work at all). After updating to ``2.2.1`` it worked correctly 👍

**Comment by joshwnj** [comment]
Looks good! we can merge this once the conflicts are resolved.

**Pull request merged**